tcms/rpc/api/testexecution.py

```diff
@@ -196,9 +196,11 @@ def update(execution_id, values, **kwargs):
             and not all_executions.filter(status__weight=0).exists()
         ):
             test_execution.run.stop_date = now
+            test_execution.run.start_date = now
             test_execution.run.save()
         elif test_execution.status.weight == 0 and test_execution.run.stop_date:
             test_execution.run.stop_date = None
+            test_execution.run.start_date = None
```

Comment:
Remove this line, we don't want to reset `start_date` after a test execution has been created.